deploy/crds/idmocp.redhat.com_v1alpha1_idm_cr.yaml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
This is another auto-generated file that should be excluded from the linter.

Reply:
Thank you removing from the linter stage this file.